src/services/cohortService.ts

```diff
@@ -40,11 +40,23 @@ export const getCohortsService = async (searchString: string) => {
   return await getCohortsQuery(searchString)
 }
 
+export const generateCohortIdService = async () => {
+  let cohortId = 1
```

Comment:
I would recommend to use `cohortNumber` or `readableId`.

Using cohortId can easily be mistaken with the id of the cohort object